lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -131,6 +149,11 @@ static bool IsStaticSwiftRuntime(Module &image) {
   return image.FindFirstSymbolWithNameAndType(swift_reflection_version_sym);
 }
 
+static bool IsStaticSwiftConcurrency(Module &image) {
+  static const ConstString task_switch_symbol("_swift_task_switch");
```

Comment:
This could be:
```suggestion
  static constexpr llvm::StringLiteral task_switch_symbol = "_swift_task_switch";
```

Reply:
The rationale here was also just to keep uniformity with the other identical helper functions in this file. But also the call `image.FindFirstSymbolWithNameAndType` requires a ConstString, so if we have a StringLiteral we would be instantiating a ConstString on every call to this method, as opposed to just once.